### PR TITLE
ui: What's new in v5.5

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/Shared/Views/Onboarding/WhatsNewView.swift [hunk 1]
```diff
@@ -314,6 +314,42 @@ private let versionDescriptions: [VersionDescription] = [
...
+                description: "With encrypted files and media."
+            ),
+            FeatureDescription(
+                icon: "link",
+                title: "Simpler interface to connect",
+                description: "Paste the link into the search bar!"
+            ),
+            FeatureDescription(
+                icon: "list.bullet.rectangle",
+                title: "Join group conversations!",
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Onboarding/WhatsNewView.swift:333]: two exclamation marks in a row
```suggestion
                description: "Paste the link into the search bar!"
            ),
            FeatureDescription(
                icon: "list.bullet.rectangle",
                title: "Join group conversations",
```

File: apps/ios/Shared/Views/Onboarding/WhatsNewView.swift [hunk 2]
```diff
@@ -314,6 +314,42 @@ private let versionDescriptions: [VersionDescription] = [
...
+                title: "Private notes",
+                description: "With encrypted files and media."
+            ),
+            FeatureDescription(
+                icon: "link",
+                title: "Simpler interface to connect",
+                description: "Paste the link into the search bar!"
+            ),
+            FeatureDescription(
+                icon: "list.bullet.rectangle",
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Onboarding/WhatsNewView.swift:332]:
```suggestion
                icon: "bubble.left.and.bubble.right",
```

File: apps/ios/Shared/Views/Onboarding/WhatsNewView.swift [hunk 3]
```diff
@@ -314,6 +314,42 @@ private let versionDescriptions: [VersionDescription] = [
...
+                title: "Join group conversations!",
+                description: "Recent history and improved [directory service](simplex:/contact#/?v=1-4&smp=smp%3A%2F%2Fu2dS9sG8nMNURyZwqASV4yROM28Er0luVTx5X1CsMrU%3D%40smp4.simplex.im%2FeXSPwqTkKyDO3px4fLf1wx3MvPdjdLW3%23%2F%3Fv%3D1-2%26dh%3DMCowBQYDK2VuAyEAaiv6MkMH44L2TcYrt_CsX3ZvM11WgbMEUn0hkIKTOho%253D%26srv%3Do5vmywmrnaxalvz6wi3zicyftgio6psuvyniis6gco6bp6ekl4cqj4id.onion)"
+            ),
+            FeatureDescription(
+                icon: "person.2",
+                title: "Safer groups",
+                description: "- block member for all to stop spam.\n- contact address in profile is not shared."
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Onboarding/WhatsNewView.swift:339]: asterisk? legacy clients still may have

Comment (spaced4ndy) [reply apps/ios/Shared/Views/Onboarding/WhatsNewView.swift:339]: also depends not only on your version, but inviting admin as well

Comment (epoberezkin) [reply apps/ios/Shared/Views/Onboarding/WhatsNewView.swift:339]: that's ok

**Pull request merged**